producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/bulkfeedback/BulkFeedbackConfigBOne.java

```diff
@@ -0,0 +1,171 @@
+package uk.gov.moj.sdt.producers.comx.config.bulkfeedback;
+
+import java.time.LocalDateTime;
+
+import com.google.common.collect.Lists;
+import org.springframework.beans.factory.config.MethodInvokingFactoryBean;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.moj.sdt.domain.BulkSubmission;
+import uk.gov.moj.sdt.producers.comx.utils.BulkFeedbackFactory;
+
+@Configuration
+@EnableAutoConfiguration
+public class BulkFeedbackConfigBOne {
+
+    @Bean("createIndividualRequestB0000001")
+    public void invokingBean() {
+        MethodInvokingFactoryBean methodInvokingFactoryBean = new MethodInvokingFactoryBean();
+        methodInvokingFactoryBean.setTargetObject(bulkFeedbackFactory());
+        methodInvokingFactoryBean.setTargetMethod("createIndividualRequests");
+
+        methodInvokingFactoryBean.setArguments(Lists.newArrayList(Lists.newArrayList("USER_REQUEST_ID_B1a",
+                                                                                     "mcolClaim",
+                                                                                     "Rejected",
+                                                                                     "DUP_CUST_REQID",
+                                                                                     "Duplicate User Request Identifier submitted USER_REQUEST_ID_B1a.",
+                                                                                     ""),
```

Comment:
Could the indentation in this section be changed to make it easier to read and not exceed the 120 characters on a line limit?  For example, change indentation of the nested Lists.newArrayList() so that it looks like this:

```
methodInvokingFactoryBean.setArguments(Lists.newArrayList(
    Lists.newArrayList("USER_REQUEST_ID_B1a",
                       "mcolClaim",
                       "Rejected",
                       "DUP_CUST_REQID",
                       "Duplicate User Request Identifier submitted USER_REQUEST_ID_B1a.",
                       ""),
    Lists.newArrayList("USER_REQUEST_ID_B1b",
                       ...,
```

Reply:
Changed indentation